vscode/src/autoedits/autoedits-provider.ts

```diff
@@ -154,6 +172,16 @@ export class AutoeditsProvider implements vscode.InlineCompletionItemProvider, v
             })
         )
 
+        const meter = metrics.getMeter('autoedit', defaultVSCodeExtensionClient().clientVersion)
+
+        this.completionLatencyMetric = meter.createHistogram<CompletionLatencyMetricAttributes>(
+            'autoedits.inlinecompletions.latency',
+            {
+                description: 'Latency of inline completions',
```

Comment:
We went through several iterations of the feature naming and landed on `autoedit`. Let's use it for consistency. Autoedit suggestions can be rendered as inline completions and/or inline decorations. To avoid confusion from the naming, we can use `suggestions` because we collect metrics for all responses, not only the ones rendered as inline completions.

```suggestion
        this.suggestionLatencyMetric = meter.createHistogram<CompletionLatencyMetricAttributes>(
            'autoedit.suggestions.latency',
            {
                description: 'Autoedit suggestions latency',
```